### ISSUE TITLE
Device "Name" resetted after system reboot

### ISSUE THREAD

**Issue by chiefjoejj** [Louisvdw/dbus-serialbattery#425]
The changeable "Name" of the bms-instance is resetted after a system reboot. Would be great if this could be fixed somehow...

**Comment by pos-ei-don** [comment]
As a workaround, I set the Name in the incl des NodeRed at startup (over mqtt)

**Comment by teefixx** [comment]
Hi Pos-ei-don
That's in general a good idea, but be careful if the VRM instances change...
see https://github.com/Louisvdw/dbus-serialbattery/discussions/481

**Comment by pos-ei-don** [comment]
Why? Mai "Main" battery is a generic mqtt device, filled by NodeRed. So no Problem with this.

**Comment by teefixx** [comment]
for the generic device it's no problem. I assume, you always assign the same VRM instance to the generic battery. I thougt, you name your real batteries as well, and there the VRM instance can change...

**Comment by pos-ei-don** [comment]
Yes, but NodeRed can "detect" the New id and assign the Name I prefer. So the id doesn't matter...

**Comment by teefixx** [comment]
is this done automatically? or do I need to have a special node for that?

**Comment by teefixx** [comment]
> As a workaround, I set the Name in the incl des NodeRed at startup (over mqtt)

@pos-ei-don 

With which node do you set the battery name and how?

**Comment by pos-ei-don** [comment]
@teefixx 

Yes, MQTT-OUT. Just write the name you wish at startup to the topic "W/[...]/CustomName" as string.
As the (at least the non ble driver) did not sent the custom field text you can enter in the BLE-APP and save on the bms, 
I used other values to "indentify" a BMS. Like Capacity. One CellPack got 280Ah, the other 281Ah.
If someone finds a better method please share.

So if is subscribe via mqtt-in to ""N/[vrm-id]/battery/+/InstalledCapacity", I get all my batteries in a seperate payload-message.
I then can easyle set the name as shown above.

**Comment by teefixx** [comment]
@pos-ei-don 
Thank you
Tried this, changed the capacity in one battery in the BMS. This value is not read out, instead VictronGX takes the value from the serial driver...
What kind of BMS do you have? I have a Daly 16S

**Comment by teefixx** [comment]
@pos-ei-don
sorry for bothering you again.
I found a way to distinct the batteries and write the values via MQTT out.
When I check it with MQTT Explorer the values are there.

<img width="407" alt="image" src="https://user-images.githubusercontent.com/35899354/233596564-95a33807-45c9-432b-ba16-3e9fb3e0e03a.png">

Still, if I have a look at the remote console of VenusOS, the names did not change. They are still "SerialBattery (Daly)"

I tried both, publish at N and W...
